pom.xml

```diff
@@ -9,17 +9,32 @@
 
     <properties>
         <java.version>1.8</java.version>
-        <handlebars-java-version>4.0.3</handlebars-java-version>
+        <handlebars-java-version>4.3.1</handlebars-java-version>
+        <handlebars-markdown-java-version>4.2.1</handlebars-markdown-java-version>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <powermock.version>1.5.6</powermock.version>
-        <dp-logging.version>v2.0.0-beta.8</dp-logging.version>
-        <fasterxml.jackson.version>2.15.0</fasterxml.jackson.version>
+        <dp-logging.version>v2.0.0-beta.10</dp-logging.version>
+        <fasterxml.jackson.version>2.16.1</fasterxml.jackson.version>
         <jetty.version>9.4.53.v20231009</jetty.version>
-        <flying-saucer.version>9.1.22</flying-saucer.version>
+        <flying-saucer.version>9.4.0</flying-saucer.version>
     </properties>
 
     <dependencies>
-
+        <dependency>
+            <groupId>ch.qos.logback</groupId>
+            <artifactId>logback-classic</artifactId>
+            <version>1.3.14</version>
+        </dependency>
+        <dependency>
+            <groupId>org.yaml</groupId>
+            <artifactId>snakeyaml</artifactId>
+            <version>1.33</version>
+        </dependency>
+        <dependency>
```

Comment:
Question: why is this new dependency needed?

Reply:
removed it as its not needed